modules/ROOT/pages/server-compatibility-collections.adoc

```diff
@@ -28,10 +28,9 @@ Couchbase Server provides the backing data store for Sync Gateway.
 
 TIP: See: {compatibility--xref} for version compatibility information.
 
-Here we provide details on how Couchbase Server's Scopes and Collections feature relates to the Couchbase mobile ecosystem.
 
-Sync Gateway offers support for Couchbase Server's default scopes and collections ({server-collections-default--xref}).
-It does not currently support named scopes or collections ({server-collections-named--xref}).
+Sync Gateway supports scopes and collections ({server-collections-default--xref}, {server-collections-named--xref} ).
```

Comment:
Haven't you just deleted `server-collections-named--xref` above?

Reply:
No, I left the links to Server as-is for now.